lib/bibliothecary/parsers/npm.rb

```diff
@@ -70,6 +70,7 @@ def self.parse_yarn_lock(file_contents)
           {
             name: dep[:name],
             requirement: dep[:version],
+            original_requirement: dep[:requirement],
```

Comment:
I'm getting a little hung up on the naming here and not loving this as it seems a little confusing. @jsonperl do you have any thoughts/suggestions?

Reply:
I have the feeling we already have a name for this in depci somewhere...

Reply:
Hmm, so it's basically the requested version in the lockfile. lockfile_requirement?